.browserlistrc

```diff
@@ -0,0 +1,7 @@
+IE >= 10
```

Comment:
Honestly, since this is run in Node only most likely we should just do Node 8 or Node 6 +....

Reply:
Since we are not in a browser

Reply:
Fair